modelci/hub/tvm_testing.py

```diff
@@ -0,0 +1,69 @@
+import tvm
+import torch
+from tvm import relay
+from pathlib import Path
+from tvm.contrib.download import download_testdata
+from PIL import Image
+import numpy 
+# import tensorflow as tf
+
+def to_tvm(model: torch.nn.Module, save_path: Path, input_shape, img_path, input_name, opt_level, override=False):
+    """
+     Convert a PyTorch nn.Module into TVM.
+    """
+    if save_path.with_suffix('.zip').exists():
+        if not override:  # file exist yet override flag is not set
+            print('Use cached model')
+            return True
+    model = model.eval()
+    try:
+        input_data = torch.randn(input_shape)
+        traced_model = torch.jit.trace(model, input_data).eval()
+    
+        # image for testing 
+        img = Image.open(img_path).resize((224, 224))
```

Comment:
we have an image pre-processing function and you can check our code (I forget where) to use it.

Reply:
Maybe you could have a reference here for the image processing
https://github.com/cap-ntu/ML-Model-CI/blob/e3713239de8404958d72fa4c0a03c1ca601ca0e4/modelci/experimental/finetuner/resnet_fine_tune_example.py#L39